.pre-commit-config.yaml

```diff
@@ -11,7 +11,6 @@ repos:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
         exclude: ^.*demographic\.000000$
-    -   id: requirements-txt-fixer
```

Comment:
intentional deletion of requirements-txt-fixed?

Reply:
Yup, poetry uses its own format, that fixer doesn't like it. Figured I'm happy to go with what poetry spits out by default.